### PR TITLE
[Artifacts] Error message when attempting to delete DatasetArtifact data including several files

### PR DESCRIPTION
**Pull request by moranbental**
update error message 
https://iguazio.atlassian.net/browse/ML-7070

### CODE REVIEW

File: server/api/crud/artifacts.py [hunk 1]
```diff
@@ -329,7 +329,7 @@ def _delete_artifact_data(
                 path
             ):
                 raise mlrun.errors.MLRunNotImplementedServerError(
-                    "Deleting artifact data of kind 'dataset' is currently supported for a single file only"
+                    "You can only delete artifact data of the kind 'dataset' if it is a single file"
```
Comment (quaark) [inline server/api/crud/artifacts.py]:
```suggestion
                    "You can only delete the data of an artifact of kind 'dataset' if it is a single file"
```

**Review by quaark — CHANGES_REQUESTED** [review]
Slight grammar fix

File: server/api/crud/artifacts.py (hunk 1 above)
Comment (moranbental) [reply server/api/crud/artifacts.py]: The error message was updated by @jillnogold

Comment (TomerShor) [reply server/api/crud/artifacts.py]: While grammatically correct, the meaning is a bit different:
- `delete artifact data of the kind 'dataset'` - means the the kind of the **data** is 'dataset'
- `delete the data of an artifact of kind 'dataset'` - means the kind of the **artifact** is 'dataset'

The second one is the correct meaning in this case.

**Review by TomerShor — APPROVED** [review]

File: server/api/crud/artifacts.py [hunk 2]
```diff
@@ -329,7 +329,7 @@ def _delete_artifact_data(
                 path
             ):
                 raise mlrun.errors.MLRunNotImplementedServerError(
-                    "Deleting artifact data of kind 'dataset' is currently supported for a single file only"
+                    "You can only delete the data of an artifact of kind 'dataset' if it is a single file"
```
Comment (alonmr) [inline server/api/crud/artifacts.py:332]: @jillnogold we want code errors to start with a verb, unlike docs it doesn't make sense to start an error message with `You can` or `You can't`. we want to say what is allowed in general without addressing the user

**Review by alonmr — CHANGES_REQUESTED** [review]
Comment for @jillnogold

File: server/api/crud/artifacts.py (hunk 2 above)
Comment (jillnogold) [inline server/api/crud/artifacts.py:332]: Aha - so the original message is perfect

Comment (jillnogold) [inline server/api/crud/artifacts.py:332]: "Deleting artifact data of kind 'dataset' is currently supported for a single file only"

**Comment by moranbental** [comment]
No need to change the error message

**Pull request closed without merging**